### ISSUE TITLE
Party names on term pages should activate filter

### ISSUE THREAD

**Issue by wrightmartin** [everypolitician/everypolitician#340]
http://everypolitician.org/uk/commons/term-table/56.html

I expected that's what it did, but it didn't

**Comment by tmtmtmtm** [comment]
Yeah, it's certainly planned that that will happen at some point, but it's non-trivial, as the filter is currently just a 'does this text appear anywhere on this card', and something like this would need to be a much more explicit check for the party (to make sure selecting "AL" works somewhere like http://everypolitician.org/germany/bundestag/term-table/11.html)

Patches welcome :)

**Comment by tmtmtmtm** [comment]
@wrightmartin why did you expect that clicking on a party would be a _filter_ (on the current page), rather than a _link_ (to a distinct Party page)?

**Comment by wrightmartin** [comment]
@tmtmtmtm I'm thinking about it, and I can't say. Considering those party names are so far away from the members, it would be difficult to show the user that something is changing way down the page. 

If I were doing this ticket and we were going down the 'filter on this page' route then I'd have to consider adjusting the design to move them closer.

Otherwise a separate page would be fine
